primitives/src/locktime/absolute.rs

```diff
@@ -406,36 +406,36 @@ mod tests {
 
     #[test]
     fn display_and_alternate() {
-        let n = LockTime::from_consensus(741521);
-        let s = format!("{}", n);
+        let lock_height = LockTime::from_consensus(741521);
+        let s = format!("{}", lock_height);
         assert_eq!(&s, "741521");
 
-        let got = format!("{:#}", n);
+        let got = format!("{:#}", lock_height);
         assert_eq!(got, "block-height 741521");
     }
 
     #[test]
     fn lock_time_from_hex_lower() {
-        let lock = LockTime::from_hex("0x6289c350").unwrap();
-        assert_eq!(lock, LockTime::from_consensus(0x6289C350));
+        let lock_time = LockTime::from_hex("0x6289c350").unwrap();
+        assert_eq!(lock_time, LockTime::from_consensus(0x6289C350));
```

Comment:
Welcome to working with locktimes, this change sounded great yesterday when we were talking about lock-by-height timelocks but now we replaced 'height' with 'time' its doesn't add much value. We probably should have used `lock_by_height` and `lock_by_time` - not to worry, its not worse than it was.